ebt/Dialog/BtSender.vala

```diff
@@ -222,6 +222,10 @@ public class BtSender : Granite.Dialog {
                 "/org/bluez/obex",
                 "org.bluez.obex.Client1"
             );
+            path_label.set_markup (GLib.Markup.printf_escaped (_("<b>From</b>: %s"), file_path.get_parent ().get_path ()));
+            device_label.set_markup (GLib.Markup.printf_escaped (_("<b>To</b>: %s"), device.name));
+            icon_label.set_from_gicon (new ThemedIcon (device.icon == null? "bluetooth" : device.icon), Gtk.IconSize.LARGE_TOOLBAR);
+            progress_label.label = _("Try connecting to %s…").printf (device.name);
```

Comment:
I think the text should be "Trying to connect to %s". The current text sounds like an instruction to do something rather than information about what is happening.